index.html

```
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="description"
+      content='RandQuote is a website to show random quotes with some features like copying the quote or saving the quote as an image and it has translation between AR and Eng.
```

HTMLHint (1.5.0)

[error] The value of attribute [ content ] must be in double quotes.

(attr-value-double-quotes)